zowe/zowe-cli
# Fix plug-in peerDep check

**What It Does**

Remove the warnings at plug-in install time that appear, even though the plug-in peerDep version-range satisfies the requirement.

**How to Test**

First you would want to reproduce the issue:
- Globally install any of the CLI versions tagged as 7.0.0-next.###
  - https://github.com/zowe/zowe-cli/tags?after=v7.0.0
- Modify a plug-in's peerDep version of imperative to `>=5.0.0-next.0 <5.0.0`.
- Install your plug-in
- Notice that there is a warning at install time

Then you can test this branch:
- Build this branch and install/link it globally
- Update the plug-in peerDep version to `>=8.0.0-next.0 <8.0.0`
- Install the plug-in
- Notice that there is no warning at install time 😋 

**Review Checklist**
I certify that I have:


**Additional Comments**

This follows the same behavior as https://semver.npmjs.com/
- Feel free to go to the website and enter the values `@zowe/imperative` and `>=5.0.0-next.0 <5.0.0` 😋 

Also, this GIF might make things easier to review:
![temp](https://github.com/zowe/zowe-cli/assets/37381190/976372dd-5b54-46f0-a894-3e2163e38f1a)
